__tests__/integration.test.ts

```diff
@@ -85,4 +90,24 @@ describe('Node host', () => {
   it.skip('can call a function from plugin with args', async () => {
     await nvim.command('e! nvimrc');
   });
+
+  it('spawns a child host if $NVIM_NODE_HOST_DEBUG is set', async () => {
+    const childHost = cp.spawn(
+      process.execPath,
+      [path.join(__dirname, '..', 'bin', 'cli.js')],
+      { env: { NVIM_NODE_HOST_DEBUG: 1 } }
+    );
+    return axios.get('http://127.0.0.1:9229/json/list').then(
+      res => {
+        childHost.kill();
+        const debuggerData = res.data[0];
+        expect(debuggerData.webSocketDebuggerUrl).toMatch(
```

Comment:
I don't know if the result of this `expect` affects the test result. I can guarantee the behaviour by removing `async` because `await` is useless for http requests.